website/docker/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:    
+  proxy: 
+    restart: unless-stopped
+    image: nginx:stable-alpine
+    volumes:
+      - static_volume:/usr/share/nginx/html
+      - ./nginx:/etc/nginx/conf.d
+    ports:
+      - 80:80
+    depends_on:
+      - web-app
+          
+  web-app:
+    restart: unless-stopped
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+    entrypoint: /app/wsgi-entrypoint.sh    
+    volumes:
+      - static_volume:/app/static
+    ports:
+      - 8000:8000
+    env_file:
+      - ../.env
+
+
+volumes:
+    static_volume:
```

Comment:
@123MwanjeMike is this compose capable of fully running the app? You might want to add postgres db servce too.